src/Util/util.h

```diff
@@ -264,7 +264,7 @@ bool end_with(const std::string &str, const std::string &substr);
 //拼接格式字符串
 template<typename... Args>
 std::string str_format(const std::string &format, Args... args) {
-#if __cplusplus > 202002L
+#if __cpp_lib_format
```

Comment:
In fact, it may be more reasonable to change it to the following. You can try it.
```
#if __cplusplus >= 202002L
   // C++20
   return std::vformat(format, std::make_format_args(args...));
```

BTW, I think there is no need to be compatible with std::format here, because after all, the string placeholder symbols of std::format are not consistent with those of snprintf.

Reply:
Works good for me